ci/terraform/production.tfvars

```diff
@@ -12,6 +12,8 @@ support_international_numbers                                       = "1"
 support_account_recovery                                            = "1"
 support_smart_agent                                                 = "0"
 client_name_that_directs_all_contact_form_submissions_to_smartagent = "di-auth-stub-relying-party-production"
+support_welsh_language_in_support_forms                             = "0"
+url_for_support_links                                               = "/contact-us"
```

Comment:
Is there a reason this is a path and the others are urls? Should this be the full url?

Reply:
Thanks @BeckaL - there is an explanation but I'm open to changing the approach. 

I'd opted to include the `url_for_support_links` variable in production so that all the environment variable keys are in place across all environments and to reduce the (admittedly very small) potential for the key to be misspelled before we go live. This is partly because I've been finding all the cognitive load of all the environment variables and AWS Secrets a bit of a challenge but also because I'm currently working on some "Go live" documentation and was thinking it's easier to say "Find and change" rather than "Add".

It should be redundant because omitting it _should_ result in the `"/contact-us"` path kicking in through the defaults. 

I'm very happy to take your view on it (this is definitely "backend" rather than "frontend" so I'm not best placed to make a call. Just let me know 👍🏽